dts/bindings/power/nuvoton,npcx-power-psl.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX Power Switch Logic (PSL) control node
```

Comment:
where is this used now?

Reply:
I have mentioned it in the first comment. Please refer the board DT file [here](https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/3698178/3/zephyr/projects/npcx_evb/npcx9/gpio.dts) in the downstream repo. 
* [CL3698178](https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/3698178) demonstrates how to use PSL input pads for minimal power consumption in board DT file.
* [CL3697271](https://chromium-review.googlesource.com/c/chromiumos/platform/ec/+/3697271) is an example to use pinctrl mechanism to configure PSL module in the drivers of downstream repo.